### ISSUE TITLE
500 Internal Server Error - file could not be downloaded

### ISSUE THREAD

**Issue by code23-isaac** [cedaro/satispress#129]
Hi,

I'm getting the following error after authentication:

`[Composer\Downloader\TransportException]                                                                                                    
  The "mydomain.com/?satispress_route=composer/packages.json" file could not be downloaded (HTTP/1.1 500 Internal Server Error  
  ) `

I'm hosting on Kinsta with a standard WP installation, routing all traffic through HTTPS.

Any ideas?

EDIT: looking at https://github.com/cedaro/satispress/blob/master/docs/composer.md#installing-packages, I wasn't asked to store the credentials after entering them, if that helps

**Comment by BrianHenryIE** [comment]
You can manually set your credentials in `~/.composer/auth.json` with the format:
```
{
    "http-basic": {
        "mywordpresssite.com": {
            "username": "fgzjS7EIfmusXrSqypYWy544Udz0qle6",
            "password": "satispress"
        }
}
```

**Comment by bradyvercher** [comment]
@BrianHenryIE Thanks for helping out!

@code23-isaac I'm not sure why you wouldn't be prompted to store the credentials (are you running the latest version of Composer), but let us know if you're still having a problem.

**Comment by smarterdigitalltd** [comment]
@code23-isaac I'm having exactly the same issue - using Kinsta too... I used to use an old version of SatisPress with no auth on Kinsta but now with the latest version, I'm getting the 500 error (although no error appears in the error log on the server...).

I've tried typing in the username (API key) and password, as well as using a local auth.json file - both result in the same thing. I can see the errors in Kinsta's access log (why it's a 500 I don't know...)

Did you get any further with the issue?

**Comment by smarterdigitalltd** [comment]
I enabled debugging on the SatisPress site, and then saw this error: https://share.getcloudapp.com/DOuxd8oX

It was related to "Pimple" that is part of a WooCommerce Product Feed plugin. All of my plugins were active, so I disabled that one, and now get a similar issue related to "Pimple" that's within SatisPress itself. 

https://share.getcloudapp.com/eDuBXxlN

I'm not entirely sure what to do with that error :-) so I might try and revert back to the old version of SatisPress (0.2.x) to get up and running.
